2019-10-16 23:24:14,523 Setting up a new session...
2019-10-16 23:24:14,578 Visdom successfully connected to server
2019-10-16 23:24:36,706 Setting up a new session...
2019-10-16 23:24:36,758 Visdom successfully connected to server
2019-10-16 23:25:22,241 Setting up a new session...
2019-10-16 23:25:22,272 Visdom successfully connected to server
2019-10-16 23:25:56,868 Setting up a new session...
2019-10-16 23:25:56,881 Visdom successfully connected to server
2019-10-16 23:26:45,512 Setting up a new session...
2019-10-16 23:26:45,525 Visdom successfully connected to server
2019-10-16 23:29:22,573 Setting up a new session...
2019-10-16 23:29:22,585 Visdom successfully connected to server
2019-10-16 23:29:43,939 Setting up a new session...
2019-10-16 23:29:43,951 Visdom successfully connected to server
2019-10-16 23:31:51,931 Setting up a new session...
2019-10-16 23:31:51,943 Visdom successfully connected to server
2019-10-16 23:31:55,497 ==>!!! Cannot find checkpoint at ./ckpts/DMFNET_MEMB3D_TRAIN/
batch_size: 1
cfg: DMFNET_MEMB3D_TRAIN
ckpt_dir: ./ckpts
criterion: generalized_dice_loss
dataset: Memb3DDataset
eps: 1.0e-05
gpu: '0'
name: DMFNET_MEMB3D_TRAIN
net: DMFNet
net_params:
  conv_channels: 128
  groups: 16
  in_channels: 2
  n_first: 32
  norm: in
  out_class: 2
num_epochs: 50
opt: Adam
opt_params:
  amsgrad: true
  lr: 0.005
  weight_decay: 1.0e-05
restore: ''
resume: ./ckpts/DMFNET_MEMB3D_TRAIN/
save_freq: 10
seed: 1024
show_image_freq: 20
show_loss_freq: 5
start_iter: 0
test_data_dir: dataset/test
test_transforms: Compose([ Pad([205, 285, 144]), Resize3D((208, 288, 144)), NumpyType((np.float32,
  np.int64)), ])
train_data_dir: dataset/train
train_list: all.txt
train_transforms: Compose([ RandCrop3D((128,128,128)), RandomRotion(10), RandomFlip(0),
  NumpyType((np.float32, np.int64)) ])
valid_data_dir: dataset/validation
valid_freq: 10
valid_list: valid.txt
weight_type: square
workers: 8

2019-10-16 23:33:01,560 Setting up a new session...
2019-10-16 23:33:01,572 Visdom successfully connected to server
2019-10-16 23:33:05,174 ==>!!! Cannot find checkpoint at ./ckpts/DMFNET_MEMB3D_TRAIN/
batch_size: 1
cfg: DMFNET_MEMB3D_TRAIN
ckpt_dir: ./ckpts
criterion: generalized_dice_loss
dataset: Memb3DDataset
eps: 1.0e-05
gpu: '0'
name: DMFNET_MEMB3D_TRAIN
net: DMFNet
net_params:
  conv_channels: 128
  groups: 16
  in_channels: 2
  n_first: 32
  norm: in
  out_class: 2
num_epochs: 50
opt: Adam
opt_params:
  amsgrad: true
  lr: 0.005
  weight_decay: 1.0e-05
restore: ''
resume: ./ckpts/DMFNET_MEMB3D_TRAIN/
save_freq: 10
seed: 1024
show_image_freq: 20
show_loss_freq: 5
start_iter: 0
test_data_dir: dataset/test
test_transforms: Compose([ Pad([205, 285, 144]), Resize3D((208, 288, 144)), NumpyType((np.float32,
  np.int64)), ])
train_data_dir: dataset/train
train_list: all.txt
train_transforms: Compose([ RandCrop3D((128,128,128)), RandomRotion(10), RandomFlip(0),
  NumpyType((np.float32, np.int64)) ])
valid_data_dir: dataset/validation
valid_freq: 10
valid_list: valid.txt
weight_type: square
workers: 8

